src/templates/next/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react"
+import { BiChevronDown } from "react-icons/bi"
+import { FilterProps } from "~/common"
+import { Heading } from "../../typography/Heading"
+
+const Filter = ({
+  filters,
+  appliedFilters,
+  setAppliedFilters,
+}: Omit<FilterProps, "type">) => {
+  const [showFilter, setShowFilter] = useState<Record<string, boolean>>({})
+
+  const updateAppliedFilters = (filterId: string, itemId: string) => {
+    const filterIndex = appliedFilters.findIndex(
+      (filter) => filter.id === filterId,
+    )
+    if (filterIndex > -1) {
+      const itemIndex = appliedFilters[filterIndex].items.findIndex(
+        (item) => item.id === itemId,
+      )
+      if (itemIndex > -1) {
+        const newAppliedFilters = [...appliedFilters]
+        newAppliedFilters[filterIndex].items.splice(itemIndex, 1)
```

Comment:
what does this do? `itemIndex` could be greater than 1 and this seems to work iff `itemIndex <= 1`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice#deletecount

believe it's just removing the specific item from the applied filter items, seems correct to me